cordova-lib/src/plugman/uninstall.js

```diff
@@ -293,58 +293,25 @@ function runUninstallPlatform(actions, platform, project_dir, plugin_dir, plugin
 function handleUninstall(actions, platform, pluginInfo, project_dir, www_dir, plugins_dir, is_top_level, options) {
     var plugin_id = pluginInfo.id;
     var plugin_dir = pluginInfo.dir;
-    var platform_modules = require('./platforms');
-    var handler = platform_modules[platform];
-    www_dir = www_dir || handler.www_dir(project_dir);
+    var handler = platform_modules.getPlatformProject(platform, project_dir);
+    www_dir = www_dir || handler.www_dir();
     events.emit('log', 'Uninstalling ' + plugin_id + ' from ' + platform);
 
+    var pluginItems = pluginInfo.getFilesAndFrameworks(platform);
     var assets = pluginInfo.getAssets(platform);
```

Comment:
Assets are not copied during installation, only during prepare (because www dir is blown on prepare). It wouldn't harm to copy them here as well, just some duplication, but they are also currently dealt with by directly calling common.asset.install(), not from the handlers with a slightly different function signature, so I left it as is for now.